easybuild/easyconfigs/s/samplot/samplot-1.3.0-foss-2021b-Python-3.9.6.eb

```diff
@@ -0,0 +1,48 @@
+# This easyconfig was written by the HPC team at Agriculture Victoria Research
+# http://agriculture.vic.gov.au/agriculture/innovation-and-research
+#
+# Author: Ben Moran <basc@agriculture.vic.gov.au>
+easyblock = 'PythonBundle'
+
+name = 'samplot'
+version = '1.3.0'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
For 2021b, and later, we no longer have Python 3 in the `versionsuffix` as we consider this the default. This is what the CI is failing on.

To change:
* remove `versionsuffix = '-Python-%(pyver)s'`
* change the filename to be `samplot-1.3.0-foss-2021b-Python-3.9.6.eb`

Reply:
Yes. Sorry for the mistype on that.